dist/components/import.html

```diff
@@ -0,0 +1,2 @@
+<link rel="import" href="/vendor/polymer/polymer.html">
+<link rel="import" href="/components/top-bar/top-bar.html">
```

Comment:
is this `/components` path correct? There's no `components` dir in the root of the style guide repo or the root of `gh-pages` app. 

Wondering under which circumstances this file would be used.